Undefined symbols:
  "saveFP", referenced from:
      __ZN9graphite27Segment7justifyEPNS_4SlotEPKNS_4FontEfNS_9justFlagsES2_S2_ in Justifier.cpp.o
      __ZN9graphite24Slot8finaliseEPKNS_7SegmentEPKNS_4FontERNS_8PositionERNS_4RectEhRf in Slot.cpp.o
  "restFP", referenced from:
      __ZN9graphite27Segment7justifyEPNS_4SlotEPKNS_4FontEfNS_9justFlagsES2_S2_ in Justifier.cpp.o
      __ZN9graphite24Slot8finaliseEPKNS_7SegmentEPKNS_4FontERNS_8PositionERNS_4RectEhRf in Slot.cpp.o
ld: symbol(s) not found

https://sourceforge.net/p/silgraphite/bugs/56/
--- src/CMakeLists.txt.orig	2013-06-11 06:55:31.000000000 -0500
+++ src/CMakeLists.txt	2013-10-05 17:22:11.000000000 -0500
@@ -123,7 +123,6 @@
 if  (${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
     set_target_properties(graphite2 PROPERTIES 
         COMPILE_FLAGS   "-Wall -Wextra -Wno-unknown-pragmas -Wendif-labels -Wshadow -Wno-ctor-dtor-privacy -Wno-non-virtual-dtor -fno-rtti -fno-exceptions -fvisibility=hidden -fvisibility-inlines-hidden -fno-stack-protector"
-        LINK_FLAGS      "-nodefaultlibs" 
         LINKER_LANGUAGE C)
     target_link_libraries(graphite2 c)
     include(Graphite)
